PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/component/ServerDependComboBox.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.connector.database.component;
+
+import com.microsoft.azure.toolkit.intellij.common.AzureComboBox;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+public class ServerDependComboBox<S, T> extends AzureComboBox<T> {
```

Comment:
weird class

Reply:
please comment other's PRs (they are split all code changes of this PR as different PRs). this PR is only drafted for testing.